README.rst

```diff
@@ -18,16 +18,32 @@ from the protobuf source IDL and additional configuration in YAML files.
 
 Installation
 ------------
+1. Optional: set up a virtualenv in your current directory:
+   .. code::
 
-You can now install artman directly from pip:
+      virtualenv env
```

Comment:
Can we change these instructions to teach `virtualenvwrapper`?

To install and make the virtualenv:

    sudo pip install virtualenv virtualenvwrapper
    mkvirtualenv --python=`which python3` artman

To use the virtualenv later:

    workon artman

This has a few advantages:

  1. Having to track where your virtualenv is located on the filesystem (as the existing instructions require) is clunky. `virtualenvwrapper` solves for this.
  2. These instructions opt you into Python 3 (which is a good thing).